src/QsCompiler/TestTargets/Libraries/Library1/Types.qs

```diff
@@ -1,11 +1,9 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-namespace Microsoft.Quantum.Testing
-{
-    open Microsoft.Quantum.Intrinsic;
+namespace Microsoft.Quantum.Testing{
```

Comment:
nit: maybe the opening bracket was inadvertently moved to the same line that the namespace declaration.

Reply:
Technically, I believe our style guide intends for them to be on the same line, though I don't think it is that prescriptive when it comes to whitespace.